go.mod

```diff
@@ -1,5 +1,11 @@
 module github.com/creditkudos/eidas
 
-go 1.11
+go 1.17
```

Comment:
This version number affects what version of go dependent modules require so we should try to keep this low for compatibility.

Reply:
It's ok to bump the docker version, but the _module_ version should stay low.